### PR TITLE
Adding allOrdersOCO endpoint

### PR DESCRIPTION
**Pull request by Amantel**
I have added description to the README file and a test. 
Test, though, I could not run - I have no OCO data in testing account.

### CODE REVIEW

File: test/auth.js [hunk 1]
```diff
@@ -79,6 +79,22 @@ const main = () => {
...
+      timestamp: new Date().getTime(),
+    })
+
+    t.true(Array.isArray(orderLists))
+    t.truthy(orderLists.length)
+
+    const [orderList] = orderLists
+
+    checkFields(t, orderList, ['orderListId', 'symbol', 'transactionTime', 'listStatusType', 'orders'])
+
```
[the remark this answers is not in the document]
Comment (Amantel) [reply test/auth.js:93]: I'm sorry?

**Comment by Amantel** [comment]
Rebased

File: test/auth.js [hunk 2]
```diff
@@ -79,6 +79,22 @@ const main = () => {
     checkFields(t, res, ['orderId', 'symbol', 'price', 'type', 'side'])
   })
 
+  test('[REST] allOrdersOCO', async t => {
+
```
[the remark this answers is not in the document]
Comment (Amantel) [reply test/auth.js]: Done

File: README.md [hunk 3]
```diff
@@ -1173,6 +1174,81 @@ console.log(
...
+  }),
+)
+```
+
+| Param      | Type    | Required | Default | Description                                               |
+|------------|---------|----------|---------|-----------------------------------------------------------|
+| timestamp  | Number  | true     |         |                                                           |
+| startTime  | Number  | false    |         |                                                           |
+| endTime    | Number  | false    |         |                                                           |
+| limit      | Integer | false    | 500     | Default Value: 500; Max Value: 1000                       |
```
[the remark this answers is not in the document]
Comment (Amantel) [reply README.md]: Done

File: test/auth.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (balthazar) [reply test/auth.js:93]: Remove the newline also ;p

File: src/http-client.js [hunk 4]
```diff
@@ -289,9 +289,13 @@ export default opts => {
     getOrder: payload => privCall('/api/v3/order', payload),
     cancelOrder: payload => privCall('/api/v3/order', payload, 'DELETE'),
 
+
+
```
Comment (balthazar) [inline src/http-client.js:293]: Remove the extra newlines here

Comment (Amantel) [reply src/http-client.js]: +

File: test/auth.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Amantel) [reply test/auth.js]: +

File: test/auth.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Amantel) [reply test/auth.js]: +

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Amantel) [reply README.md]: +

**Comment by balthazar** [comment]
Looks good, thanks!

**Pull request merged**